tests/page_objects/employee_health_portal_page.yaml

```diff
@@ -22,3 +24,21 @@ confirmation_modal:
   email_address: //*[@id='dialogs']//p/strong/text()
   close_modal_button: //*[@id='dialogs']//button
 
+home_page:
```

Comment:
Lets continue with the same name convention of adding employee_health before the page name